admin/system/edit_produk.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+    include("connection.php");
+    session_start();
+
+    //get data
+    $id_produk = $_GET["id"];
+    $nama_produk = $_POST["nama_produk"];
+    $jenis_produk = $_POST["jenis_produk"];
+    $kategori = $_POST["kategori"];
```

Comment:
Codacy found an issue: [Detected usage of a possibly undefined superglobal array index: $_POST["kategori"]. Use isset() or empty() to check the index exists before using it](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5762293)